docker-compose.yaml

```diff
@@ -0,0 +1,14 @@
+services:
+  app:
+    build:
+      context: .
+    command: bundle exec rails server --bind 0.0.0.0
+    ports:
+      - 3000:3000
+    volumes:
+      - .:/var/app
+      - ~/Music:/var/music
+      - bundle_cache:/bundle
```

Comment:
What's the difference between those three `volume`s definition, and why only `bundle_cache` is defined in the YAML?

Reply:
@ramos-ph The first two are simple `bind` mounts (see https://docs.docker.com/compose/compose-file/05-services/#volumes). We are mapping the source code of the app to /var/app for it to run, and ~/Music for the default music directory.

I believe `bundle_cache` is defined as a volume as to not force us to download the bundle gems everytime. This is used because by default `docker-compose run` creates a separate container, which is not the same as the app container.